conan/cli/commands/graph.py

```diff
@@ -298,3 +301,109 @@ def graph_explain(conan_api, parser,  subparser, *args):
 
     ConanOutput().title("Closest binaries")
     return {"closest_binaries": pkglist.serialize()}
+
+
+def outdated_text_formatter(result):
+    cli_out_write("======== Outdated dependencies ========", fg=Color.BRIGHT_MAGENTA)
+
+    for key, value in result.items():
```

Comment:
The ``key`` is not used? Seems then the chose of key is not very relevant?

Reply:
If you activate  a linter, it will warn you about unused variables and other issues